test/internet/test-dns-cares-domains.js

```diff
@@ -20,8 +21,8 @@ const methods = [
 methods.forEach(function(method) {
   const d = domain.create();
   d.run(function() {
-    dns[method]('google.com', function() {
+    dns[method](addresses.INET_HOST, common.mustCall(() => {
```

Comment:
Improving the test at the same time? Nice!

Reply:
@gibfahn There are so many missing `common.mustCall` in the internet tests...better leave it to other PRs but this is in a loop, so I thought why not :D..